pyvista/core/grid.py

```diff
@@ -161,16 +162,23 @@ def _from_arrays(self, x: np.ndarray, y: np.ndarray, z: np.ndarray):
             Coordinates of the points in z direction.
 
         """
+
+        def raise_not_unique(arr, name):
```

Comment:
Some minor comments:
1. This is OK here, but it could be a generally useful method.
2. Do we need the `name` argument here?  Won't the stack trace be enough?  Also the variable name is x in this private method, but the user supplied variable could be anything.

Reply:
Both resolved in 3b859099107f76c79878254a4b9952b9ebec7731.